### PR TITLE
Update strings.xml: Deutsche Übersetzung

### PR DESCRIPTION
**Pull request by swissalpine**

### CODE REVIEW

**Comment by jotomo** [comment]
Thanks!

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -775,4 +775,21 @@
     <string name="openapsama_link_to_preferncejson_doc_txt">Achtung! Normalerweise musst Du diese Werte nicht ändern. Bitte KLICKE HIER und LESE den Text. Verändere Werte erst, wenn Du den Inhalt des Textes verstanden hast.</string>
     <string name="combo_actvity_reading_basal_profile">Basalratenprofil wird gelesen</string>
     <string name="pump_basebasalrate">%.2f IE/h</string>
+    <string name="ro_lang">Rumänisch</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: This should not be translated.

**Comment by AdrianLxM** [comment]
@jotomo  unmerge or edit?

**Comment by jotomo** [comment]
I'll remove it, doing some editing there anyway.

**Comment by swissalpine** [comment]
Sorry, didn't know that.

**Pull request merged**